package.json

```diff
@@ -134,7 +134,7 @@
     "ngx-device-detector": "^1.3.0",
     "ngx-md": "^7.0.0",
     "ngx-pagination": "^3.0.3",
-    "node-sass": "^4.6.1",
+    "node-sass": "^4.14.0",
```

Comment:
This is new since my initial review... was it deliberate and what's it for? It's resulted in quite a few changes in the `yarn.lock` file. 🤔